mvc/src/main/java/nextstep/mvc/DispatcherServlet.java

```diff
@@ -34,36 +33,35 @@ public void addHandlerMapping(HandlerMapping handlerMapping) {
         handlerMappings.add(handlerMapping);
     }
 
+    public void addHandlerAdapter(HandlerAdapter handlerAdapter) {
```

Comment:
```suggestion
    public void addHandlerAdapter(HandlerAdapter... handlerAdapter) {
```

지금은 HandlerAdapter를 여러 개 등록하려면, 메소드 호출을 여러 번 해야하는데요.
메소드를 가변인자로 변경하면 메소드 호출을 한 번만 해도 돼요!